.github/actions/maximize-build-space/action.yml

```diff
@@ -0,0 +1,206 @@
+# https://github.com/easimon/maximize-build-space/blob/fadc013e293a3453768b4ddb9db8c85104752807/action.yml
```

Comment:
Can we not request approval to run this action at the organization level?

If there is an issue tracking that, maybe flag that in a comment here.

Reply:
I can, but I get the feeling that the sec team is really antsy about allowing actions that are under the control of a single person.